BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -1711,6 +1724,10 @@ IEnumerator BoostRoutine()
                 {
                     audioSource.Stop();
                 }
+                if (decoupleBoosters)
+                {
+                    if(burnRate > 0)part.mass -= burnRate; //deduct the fuel from the mass
```

Comment:
part mass changes are done via IPartMassModifier; this won't do anything.
The boostermass using the same code might have worked at one point in KSP's past, but not in 1.8x+

Reply:
I will read more about the IPartMassModifier documentation so i can adapt the code about that